Tasks/KubernetesManifestV0/src/actions/promote.ts

```diff
@@ -13,9 +13,20 @@ export async function promote(ignoreSslErrors?: boolean) {
     const kubectl = new Kubectl(await utils.getKubectl(), TaskInputParameters.namespace, ignoreSslErrors);
 
     if (canaryDeploymentHelper.isCanaryDeploymentStrategy()) {
+        if (canaryDeploymentHelper.isTrafficSplitCanaryStrategy()) {
+            tl.debug('Redirecting traffic to canary deployment.');
+            canaryDeploymentHelper.adjustTraffic(kubectl, TaskInputParameters.manifests, 0, 0, 1000);
```

Comment:
What % of traffic are we redirecting to Canary ? Can we add that information also in the debug message?